download_rpms.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+'''Download correct NVRs for python-versions to operate on.'''
+
+import sys
+import logging
+from libtaskotron.directives import koji_directive
+
+def download_rpms(koji_build, rpmsdir, arch=['x86_64'], arch_exclude=[],
+                  src=True, debuginfo=False, build_log=True):
+    '''Download RPMs for a koji build NVR.'''
+
+    koji = koji_directive.KojiDirective()
+
+    print 'Downloading rpms for %s into %s' % (koji_build, rpmsdir)
```

Comment:
It's 2018 and you've just wrote a Python 2 only script. Every time you do that, some little part in me dies. We are **not** going to maintain a Legacy Python only script in our repo. It's already tedious to keep in mind that our code here needs to be Python 2 compatible, this is just making it worse. I suppose this code should be part of taskotron, not our repo, but I'll get to this in a different section.

Reply:
That's because I'm not familiar with Python 3, even in 2018 :-/ This code is a semi-temporary file that I've already copied to several tasks and I'm definitely not happy about it. It's there because:
a) invoking `koji download-build` command directly is not sufficient for this task (you need `build.log`s)
b) you can't interact with python directly in ansible, so you need a wrapper that accepts cmdline arguments and converts them into python calls which use our `koji_directive` helper library

In the future, we'd like to have a native ansible module that exposes all important input arguments, so that you can call it directly and don't need to have a wrapper.

Of course, if you decide to convert this to python23 in the meantime (or even a better idea), I have no issue with it (quite the opposite).